### PR TITLE
Troubleshooting method for boot issues 

### PR DESCRIPTION
**Pull request by aervnu**

### CODE REVIEW

File: src/wiki/troubleshooting.md [hunk 1]
```diff
@@ -136,3 +136,41 @@ You must have a Windows ISO beforehand to do this. If you haven't got one or are
...
+
+![](./img/troubleshooting/bios.png)
+
+::: tip Note
+The `bootsect` command works with different types of Windows NT bootloaders. Windows NT 3.1 up to XP and Server 2003 (NT 5.0-5.2) use NTLDR to load the OS, as such the `nt52` flag must be used. Vista (NT 6.0+) and above use BOOTMGR instead of NTLDR, meaning `nt60` must be used instead of `nt52`.
+:::
+
+### UEFI
+
+Fortunately things are much more modular than BIOS over here, and this troubleshooting method has a 100% success rate for the computer to get past the bootloader phase. Similarly, boot an installation disk and start a command prompt.
```
Comment (Erisa) [inline src/wiki/troubleshooting.md:162]: Well its not 100% if something like the partition table or efi partition are screwed up, or if there are undiagnosed hardware issues as the guide mentions at the beginning. Probably more accurate to say it's "highly likely" to succeed.

Comment (aervnu) [reply src/wiki/troubleshooting.md:162]: Mmk lemme fix that, sec

**Review by Erisa — APPROVED** [review]

**Comment by Erisa** [comment]
The content looks good, don't forget to add a link to the list:
https://github.com/Microsoft-community/Community-website/blob/master/.vitepress/config.ts#L82

**Comment by aervnu** [comment]
> The content looks good, don't forget to add a link to the list: https://github.com/Microsoft-community/Community-website/blob/master/.vitepress/config.ts#L82

Bit confusing there. Doesn't that file contain the links on the dropdown menu on the homepage?

**Comment by Erisa** [comment]
It populates both the navigation menu and the list of wikis on the wiki home page.

**Comment by Erisa** [comment]
My bad, thought it was a new page. All good then.

**Review by FloatingMilkshake — APPROVED** [review]

**Pull request merged**